payu/models/access.py

```diff
@@ -52,13 +52,24 @@ def __init__(self, expt, name, config):
             if model.model_type == 'cice5':
                 model.access_restarts.append(['u_star.nc', 'sicemass.nc'])
 
+            if model.model_type == 'cice':
+                # Structure of model coupling namelist
+                model.cpl_fname = 'input_ice.nml'
+                model.cpl_group = 'coupling'
+                model.runtime0_key = 'runtime0'
+                model.start_date_nml_name = "restart_date.nml"
```

Comment:
Should keys for "init_date", "inidate" and "runtime" be here rather than hardcoded at line 182 ?

Have we documented what each field means somewhere?

Reply:
That's a good idea – makes it more consistent. I've added in these extra keys, and included comments briefly specifying what their meaning:

https://github.com/ACCESS-NRI/payu/blob/c19133c83109e768236486b0879d09e6f05c3be2/payu/models/access.py#L60-L67

Reply:
Nice